grading/main_runner.cpp

```diff
@@ -14,9 +14,85 @@
 // =====================================================================
 // =====================================================================
 
+std::string addRedirectsToCommand(std::string this_command, std::string which){
+  // Check to see if the instructor is already capturing STDIN
+  if(this_command.find("1>") == std::string::npos){
```

Comment:
I think this (and other find's in this function) is fragile to quoted or escaped characters on the command line.
I'm not sure why so much additional code is needed(?)

Reply:
Was this totally not implemented before?  (it may not have been)